js_modules/dagit/packages/core/src/runs/RunStatusDots.tsx

```diff
@@ -27,7 +27,7 @@ interface RunStatusProps {
 }
 
 export const RunStatusIndicator: React.FC<RunStatusProps> = React.memo(({status, size}) => {
-  if (status === 'STARTED') {
+  if (status === 'STARTED' || status === 'CANCELING') {
     return <Spinner purpose="caption-text" />;
```

Comment:
We could make the spinner red in the canceling case, though I don't. know if it might look terrible or if @braunjj might hate it. 😆